src/browser.mts

```diff
@@ -1,65 +1,53 @@
 import { Fetcher } from './fetcher/fetcher.mjs';
 import { FetchConfig } from './fetcher/types.mjs';
 import { paths } from './types.mjs';
+import * as Sentry from '@sentry/browser';
+import { CaptureConsole } from '@sentry/integrations';
+import { normalizeEnvironmentName } from './utils.mjs';
 
-// This is a hack to forward console.log to the parent window.
-const setupLogForwarding = (apiUrl: string) => {
-  if (!window.parent) {
-    console.warn('No parent window found. Not forwarding logs.');
-    return;
-  }
-  const origin = new URL(apiUrl).origin;
-
-  const proxiedMethods = [
-    'log',
-    'error',
-    'info',
-    'trace',
-    'warn',
-    'debug',
-  ] as const;
-
-  console = new Proxy(console, {
-    get: function (target, prop, receiver) {
-      const method = prop as typeof proxiedMethods[number];
-      if (proxiedMethods.includes(method)) {
-        return function (...rest: any[]) {
-          // window.parent is the parent frame that made this window
-          window.parent.postMessage(
-            {
-              source: 'iframe',
-              message: {
-                type: prop,
-                payload: rest,
-              },
-            },
-            origin
-          );
-          target[method].apply(rest);
-        };
-      }
-      return Reflect.get(target, prop, receiver);
-    },
-  });
-};
-
-const PlaytBrowserClient = ({ apiUrl }: { apiUrl: string }) => {
-  setupLogForwarding(apiUrl);
-
+const PlaytBrowserClient = ({
+  gameId,
+  apiUrl,
+}: {
+  gameId: string;
+  apiUrl: string;
+}) => {
   const fetcher = Fetcher.for<paths>();
   const config: FetchConfig = {
     baseUrl: apiUrl,
     use: [],
+    init: {
+      headers: {
+        'User-Agent': `playt-browser-client/${process.env.npm_package_version}`,
```

Comment:
is this going to become static during build?

Reply:
yea, Babel plugin. I'm currently trying to figure out whether the headers arrive properly

Reply:
ok so as I already kind of feared, in the browser client `fetch`, the user-agent is ignored. backend client works

Reply:
I don't understand why though because the fetch spec says it's allowed to change since like 2015 or so :(

Reply:
Ah, it's Chromium that has still not implemented it. FF works. Well, should be good enough to get it from some browsers to see what client versions are still in usage. Also we always still get the server client version which should always be using the same one anyway. And other browsers might fix it in the future.